GitUI/Script/ScriptRunner.cs

```diff
@@ -134,7 +134,8 @@ private static CommandStatus RunScriptInternal(IWin32Window owner, IGitModule mo
                 command = command.Replace(NavigateToPrefix, string.Empty);
                 if (!string.IsNullOrEmpty(command))
                 {
-                    string revisionRef = new Executable(command, module.WorkingDir).GetOutputLines(argument).FirstOrDefault();
+                    ExecutionResult result = new Executable(command, module.WorkingDir).Execute(argument);
+                    string revisionRef = result.StandardOutput.Split('\n').FirstOrDefault();
```

Comment:
```suggestion
                    string revisionRef = result.StandardOutput.LazySplit('\n').FirstOrDefault();
```

Reply:
Not available here

Reply:
We'll need to see whether we always split it, and maybe move the split into `ExecuteAsync` changing `ExecutionResult` to something similar:
```cs
    public readonly struct ExecutionResult
    {
        public string[] StandardOutput { get; }
        public string[] StandardError { get; }
```